nengo/spa/actions.py

```diff
@@ -154,7 +154,11 @@ class Actions(object):
     def __init__(self, *args, **kwargs):
         self.actions = None
         self.args = args
-        self.kwargs = kwargs
+        self.kwargs = OrderedDict(kwargs)
```

Comment:
That said, this was a problem before, too. At least this change makes things a *bit* better, but we should definitely put a comment in that things are not as ordered as they look. There's a PEP request for [ordered kwargs](https://www.python.org/dev/peps/pep-0468/), but without something like that, I don't think we can get the ordering without changing the syntax. The other option is to sort the keys (either as we add them, or when we're building the model), so that at least it's deterministic, if not in the same order that the user added them.